Change custom plugin directory from /nodes to /custom (#1664)
n8n-io/n8n-docs: _snippets/integrations/creating-nodes/testing.md

  ```

    !!! note "Check your directory"
        Make sure you run `npm link <node-name>` in the nodes directory within your n8n installation. This can be: 

>         * `~/.n8n/custom/`

Reviewer: ```suggestion
            * `~/.n8n/custom/`
```

>         * `~/.n8n/<your-custom-name>`: if your n8n installation set a different name using `N8N_CUSTOM_EXTENSIONS`.

Reviewer: ```suggestion
            * `~/.n8n/<your-custom-name>`: if your n8n installation set a different name using `N8N_CUSTOM_EXTENSIONS`.
```

  ```

    !!! note "Check your directory"

>         Make sure you run `npm link <node-name>` in the nodes directory within your n8n installation. This is probably something like `~/.n8n/custom/`

Reviewer: ```suggestion
        Make sure you run `npm link <node-name>` in the nodes directory within your n8n installation. This can be: 
        * `~/.n8n/custom/`
        * `~/.n8n/<your-custom-name>`: if your n8n installation set a different name using `N8N_CUSTOM_EXTENSIONS`.
```
